src/modules/OpenChannel/context/utils.ts

```diff
@@ -21,22 +21,29 @@ export const shouldFetchMore = (messageLength: number, maxMessages?: number): bo
 };
 
 /* eslint-disable default-param-last */
-export const scrollIntoLast = (initialTry = 0, scrollRef: React.RefObject<HTMLElement>): void => {
+export const scrollIntoLast = (
+  initialTry = 0,
+  scrollRef: React.RefObject<HTMLElement>,
+): void => {
   const MAX_TRIES = 10;
-  const currentTry = initialTry;
-  if (currentTry > MAX_TRIES) {
+
+  if (initialTry > MAX_TRIES) {
```

Comment:
Is there any reason why the name of this variable is changed from `currentTry` to `initialTry`?

Reply:
I thought there's no advantage for renaming `initialTry` to `currentTry`, so I reduced it;;